backend/models/EventParticipation.js

```diff
@@ -0,0 +1,26 @@
+// create a user moongose model with email,username and password
+const mongoose = require("mongoose");
+
+const eventParticipationSchema = new mongoose.Schema({
+  eventId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Event",
+  },
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+
+  status: {
+    type: String,
+    enum: ["pending", "accepted", "denied"],
+    default: "pending",
+  },
+});
+
+const EventParticipation = mongoose.model(
```

Comment:
Be sure to look into the unique index from MongoDB. There can never be a duplicate of this tuple (Event ID, User ID) as it would not make sense that a user can attend the same event TWICE